Makefile

```diff
@@ -11,7 +11,8 @@ test: install | emily-client-source
 
 integration-test: install | emily-client-source
 	docker compose --file docker-compose.test.yml up --detach
-	cargo test --all-features
+	cargo test
+	cargo test --test '*' --all-features -- --test-threads=1
```

Comment:
Why do we need to run the integration tests sequentially? I'd hope that they would be sufficiently isolated so that they still can run in parallel, even if they share common resources.

Reply:
The RBF tests depend on a block *not* being created until a specific time in order to test out the functionality. Specifically, in order to know if the network would accept an RBF transaction, the transaction that it is replacing must not already be confirmed. But all of the bitcoind integration tests need to generate a block in order to run correctly, so they cannot run in parallel with the RBF tests.

Reply:
Okay yeah that sounds more difficult to parallelize. Then I'm fine with having this being sequential for the time being.